Command/StatusCommand.php

```diff
@@ -70,7 +63,7 @@ public function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
-        $table->render();
+        $table->render($output);
```

Comment:
Something like

````php
if (class_exists('Symfony\Component\Console\Table`) {
  // ...
} else {
  // ...
}
````

or we can keep the deprecation and just make a new release for 3.0 which drops 2.3 support